### PR TITLE
Updating m3us redux

### PR DESCRIPTION
**Pull request by AntiPontifex**

### CODE REVIEW

File: streams/ca.m3u [hunk 1]
```diff
@@ -133,11 +133,7 @@ http://dee7mwgg9dzvl.cloudfront.net/hls/uvagut/playlist.m3u8
 https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="WCGtv.ca" status="online",WCGtv Public (720p)
 https://wowzastream.westmancom.com/wcgtvlive/wcgtvPSA.stream/master.m3u8
-#EXTINF:-1 tvg-id="" status="online",Azstar TV (1080p) [Not 24/7]
```
Comment (LaneSh4d0w) [inline streams/ca.m3u:136]: Why deleting this channel if it works ?

File: streams/ca.m3u [hunk 2]
```diff
@@ -133,11 +133,7 @@ http://dee7mwgg9dzvl.cloudfront.net/hls/uvagut/playlist.m3u8
...
-http://live.livestreamtv.ca/azstar/smil:azstar.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",BNN Bloomberg
+#EXTINF:-1 tvg-id="BNNBloomberg.ca" status="online",BNN Bloomberg
 https://pe-ak-lp01a-9c9media.akamaized.net/live/News1BNNDigi/p/dash/00000001/1bba52dc66e4c68e/manifest.mpd
-#EXTINF:-1 tvg-id="" status="online",Ontario Parliamentary Network (FR) (720p)
```
Comment (LaneSh4d0w) [inline streams/ca.m3u:140]: Same question here.

Comment (AntiPontifex) [reply streams/ca.m3u:140]: Duplicate: up to you.

File: streams/ca.m3u (hunk 1 above)
Comment (AntiPontifex) [reply streams/ca.m3u:136]: Duplicate: up to you.

**Comment by AntiPontifex** [comment]
Also, may I be given the permissions needed to add labels. Thank you.

File: streams/ca.m3u [hunk 3]
```diff
@@ -132,6 +132,6 @@ http://dee7mwgg9dzvl.cloudfront.net/hls/uvagut/playlist.m3u8
 #EXTINF:-1 tvg-id="VBS.ca" status="online",VBS (480p) [Not 24/7]
 https://uni6rtmp.tulix.tv/vbstv/vbsabr.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="WCGtv.ca" status="online",WCGtv Public (720p)
-https://wowzastream.westmancom.com/wcgtvlive/wcgtvPSA.stream/master.m3u8
+https://wowzastream.<<<<<<< master
```
Comment (LaneSh4d0w) [inline streams/ca.m3u:135]:
```suggestion
https://wowzastream.westmancom.com/wcgtvlive/wcgtvPSA.stream/master.m3u8
```

File: streams/ba.m3u [hunk 4]
```diff
@@ -24,4 +24,4 @@ https://prd-hometv-live-open.spectar.tv/ERO_1_083/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Super TV Media
 https://mirtv.club/live/mirtv/index.m3u8
 #EXTINF:-1 tvg-id="RTRSTV.ba" status="online",РТРС (576p) [Geo-blocked]
-https://parh.rtrs.tv/tv/live/playlist.m3u8
+https://parh.rtrs.tv/tv/live/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/ba.m3u:27]:
```suggestion
https://parh.rtrs.tv/tv/live/playlist.m3u8
```

Comment (AntiPontifex) [reply streams/ba.m3u:27]: There's no change here.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

File: streams/ca.m3u [hunk 5]
```diff
@@ -133,11 +133,5 @@ http://dee7mwgg9dzvl.cloudfront.net/hls/uvagut/playlist.m3u8
...
 https://wowzastream.westmancom.com/wcgtvlive/wcgtvPSA.stream/master.m3u8
-#EXTINF:-1 tvg-id="" status="online",Azstar TV (1080p) [Not 24/7]
-http://live.livestreamtv.ca/azstar/smil:azstar.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",BNN Bloomberg
+#EXTINF:-1 tvg-id="BNNBloomberg.ca" status="online",BNN Bloomberg
 https://pe-ak-lp01a-9c9media.akamaized.net/live/News1BNNDigi/p/dash/00000001/1bba52dc66e4c68e/manifest.mpd
-#EXTINF:-1 tvg-id="" status="online",Ontario Parliamentary Network (FR) (720p)
-http://origin-http-delivery.isilive.ca/live/_definst_/ontla/house-fr/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",WATAN-E-MAA
-https://5caf24a595d94.streamlock.net:1937/8132/8132/playlist.m3u8
```
Comment (Alstruit) [inline streams/ca.m3u:143]: Please explain the removal of WATAN E-MAA.

Comment (AntiPontifex) [reply streams/ca.m3u:143]: As I explained before, it was a duplicate entry. If you want it put back in, by all means.

**Review by AntiPontifex — COMMENTED** [review]
Fine by me.

**Review by AntiPontifex — COMMENTED** [review]
Again, fine by me.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by Alstruit — DISMISSED** [review]

**Comment by AntiPontifex** [comment]
You want to leave the name in Cyrillic? Ok by me,

**Review by Alstruit — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by AntiPontifex — COMMENTED** [review]
So be it; I think I may have finally figured this github thing out. Thank you both for your patience and your help.

**Pull request merged**